### PR TITLE
[cli] allow specify OT executable when adding nodes

### PR DESCRIPTION
**Pull request by simonlingoogle**
This PR allows users to specify a different OT executable when adding nodes instead of the default executable configured by `-ot-cli <executable>`.

### CODE REVIEW

File: cli/CmdRunner.go [hunk 1]
```diff
@@ -226,6 +226,10 @@ func (rt *CmdRunner) executeAddNode(cc *CommandContext, cmd *AddCmd) {
 		cfg.RadioRange = cmd.RadioRange.Val
 	}
 
+	if cmd.Executable != nil {
+		cfg.Executable = cmd.Executable.Executable
```
Comment (wgtdkp) [inline cli/CmdRunner.go:230]:
```suggestion
		cfg.ExecutablePath = cmd.Executable.Path
```

Comment (simonlingoogle) [reply cli/CmdRunner.go]: accepted

**Comment by jwhui** [comment]
@simonlingoogle , please help resolve conflict.

**Comment by simonlingoogle** [comment]
> @simonlingoogle , please help resolve conflict.

resolved

**Pull request merged**